.gitignore

```diff
@@ -2,4 +2,5 @@
 .python-version
 *.pyc
 *.anki2
-.pytest_cache
\ No newline at end of file
+.pytest_cache
+meta.json
```

Comment:
what is `meta.json`?

Reply:
it's added by Anki (2.1 at least), and contains the addon name and if it's enabled by the addon manager; I believe it also stores the default config file values if those are used. 

I've added it to the git ignore because it's maintained by the app, and any change in the 'enabled' value would be recorded as a change.

see here: [Anki 2.1 manual](https://apps.ankiweb.net/docs/addons.html#add-on-folders)